components/image/picker/src/index.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import ImagePlaceholder from '@schibstedspain/sui-image-placeholder'
+
+class ImagePicker extends Component {
+  constructor (props) {
+    super()
+
+    const { images } = props
+    let filteredImages = []
+    let selected = -1
+
+    if (images.length > 0) {
+      filteredImages = images.map(image => {
+        if (typeof image.src !== 'undefined' && typeof image.thumb !== 'undefined') { return image }
+        return false
+      }).filter(image => image !== false)
+
+      if (filteredImages.length > 0) {
+        selected = filteredImages.findIndex(image => image.selected === true)
+        selected = (selected === -1) ? 0 : selected
+      }
+    }
+
+    this.state = {
+      selected,
+      images: filteredImages
+    }
+  }
+
+  _getHeadImage () {
+    const { images, selected } = this.state
+    const { emptyImage } = this.props
+
+    const placeHolder = {
+      alt: 'Placeholder',
+      src: emptyImage
+    }
+
+    const src = (images.length === 0) ? emptyImage : images[selected].src
+    const alt = (images.length === 0) ? 'No text available' : images[selected].alt
+
+    return (
+      <div className='sui-ImagePicker-image--vcenter'>
+        <div className='sui-ImagePicker-image--hcenter'>
+          <ImagePlaceholder src={src} alt={alt} placeholder={placeHolder} />
+        </div>
+      </div>
+    )
+  }
+
+  _getListImages () {
+    const { images, selected } = this.state
+
+    const imageList = images.map((image, index) => {
+      const itemClass = cx('sui-ImagePicker-thumbItem', {
+        'sui-ImagePicker-thumbItem--selected': index === selected
+      })
+      return (
+        <div className={itemClass} key={image.src} onClick={() => this._handleClick(image)}>
```

Comment:
if you wrap with `ul`, use `li`, not `div`

Reply:
Good point too. Legacy ideas.